code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -69,6 +69,18 @@
 		return
 	key = user.key
 
+/mob/living/simple_animal/hostile/poison/giant_spider/araneus
+	name = "Sergeant Araneus"
+	desc = "A fierce companion for any person of power, this spider has been carefully trained by NanoTrasen specialists. Its beady, staring eyes send shivers down your spine."

```

Comment:
Official living lore indicates it's Nanotrasen

https://github.com/tgstation/-tg-station/search?utf8=%E2%9C%93&q=nanotrasen